packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-load-balancer.ts

```diff
@@ -458,6 +460,11 @@ export interface IApplicationLoadBalancer extends ILoadBalancerV2, ec2.IConnecta
    */
   readonly vpc?: ec2.IVpc;
 
+  /**
+   * The IP Address Type for this load balancer
+   */
+  readonly ipAddressType?: IpAddressType;
```

Comment:
include an `@default` doc so consumer can easily figure out behaviour when unset

```suggestion
   *
   * @default IpAddresstype.IPV4
   */
  readonly ipAddressType?: IpAddressType;
```